modules/xfem/src/base/XFEMFuncs.C

```diff
@@ -814,4 +814,131 @@ crossProduct2D(const Point & point_a, const Point & point_b)
   return (point_a(0) * point_b(1) - point_b(0) * point_a(1));
 }
 
+Real
+pointSegmentDistance(const Point & x0, const Point & x1, const Point & x2, Point & xp)
+{
+  Point dx = x2 - x1;
+  Real m2 = dx * dx;
+  // find parameter value of closest point on segment
```

Comment:
By 'parameter value' do you mean 'parametric coordinate'?

Reply:
correct. change is made.